### PR TITLE
reference/configuration: update configurations related to the unified thread pool

### PR DESCRIPTION
**Pull request by sticnarf**
### What is changed, added or deleted? (Required)

Make the description of the default value of `unify-read-pool` more precise.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v4.0 changes for now)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from https://github.com/pingcap/docs/pull/2063
- Other reference link(s):
The documentation is changed due to https://github.com/tikv/tikv/pull/7059 and https://github.com/tikv/tikv/pull/7234

### CODE REVIEW

File: reference/configuration/tikv-server/configuration-file.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 ### `unify-read-pool`
 
 + 是否使用单个线程池处理所有的读请求。
-+ 默认值：true
++ 默认值：如果 `readpool.storage` 和 `readpool.coprocessor` 中没有自定义配置，则该选项默认值为 true，否则为 false.
```
Comment (TomShawn) [inline reference/configuration/tikv-server/configuration-file.md]:
```suggestion
+ 默认值：如果 `readpool.storage` 和 `readpool.coprocessor` 中没有自定义配置，则该选项默认值为 true。否则默认值为 false。
```

Comment (sticnarf) [reply reference/configuration/tikv-server/configuration-file.md]: Changed.

**Comment by TomShawn** [comment]
@sticnarf Please involve a technical review, thanks!

**Comment by sticnarf** [comment]
I've updated this PR with latest configuration format changes in TiKV. Please take a look again @TomShawn

File: reference/configuration/tikv-server/configuration-file.md [hunk 2]
```diff
@@ -129,7 +119,11 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 ## readpool.storage
 
 存储线程池相关的配置项。
-以下配置只在 `unify-read-pool` 是 `false` 时生效。
+
+### `use-unified-pool`
+
++ 是否使用统一的读取线程池（在 readpool.unified 中配置）处理存储请求。该选项为 false 时，使用单独的存储线程池，通过本节中的其余配置项配置。
```
Comment (TomShawn) [inline reference/configuration/tikv-server/configuration-file.md:125]:
```suggestion
+ 是否使用统一的读取线程池（在 [`readpool.unified`](#readpoolunified) 中配置）处理存储请求。该选项值为 false 时，使用单独的存储线程池。通过本节中的其余配置项配置单独的线程池。
```

File: reference/configuration/tikv-server/configuration-file.md [hunk 3]
```diff
@@ -176,9 +170,12 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 
 ## readpool.coprocessor
 
-coprocessor 线程池相关的配置项。
+协处理器线程池相关的配置项。
+
+### `use-unified-pool`
 
-以下配置只在 `unify-read-pool` 是 `false` 时生效。
++ 是否使用统一的读取线程池（在 readpool.unified 中配置）处理协处理器请求。该选项为 false 时，使用单独的协处理器线程池，通过本节中的其余配置项配置。
```
Comment (TomShawn) [inline reference/configuration/tikv-server/configuration-file.md:177]: ditto

File: reference/configuration/tikv-server/configuration-file.md [hunk 4]
```diff
@@ -176,9 +170,12 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
++ 默认值：如果本节中没有其他配置，默认为 true。否则，为了升级兼容性，默认为 false，请根据需要更改 readpool.unified 中的配置后再启用该选项。
```
Comment (TomShawn) [inline reference/configuration/tikv-server/configuration-file.md:178]:
```suggestion
+ 默认值：如果本节 (`readpool.storage`) 中没有其他配置，默认为 true。否则，为了升级兼容性，默认为 false，请根据需要更改 (#readpoolunified) 中的配置后再启用该选项。
```

**Comment by sticnarf** [comment]
Thanks @TomShawn . I've changed it as you suggest.

**Pull request merged**